utils/funcexp/functor_str.h

```diff
@@ -120,7 +120,8 @@ class Func_Str : public Func
                 break;
 
             default:
-                return fp->data()->getStrVal(row, isNull);
+				fFloatStr = fp->data()->getStrVal(row, isNull);
+                return;
                 break;
```

Comment:
can remove the break

Reply:
That generally isn't a good idea. If a later date we remove the return (may not be likely, but hypothetical) we may missing adding the break back. Better to let the compiler realise that.